src/components/Agreements/ViewAgreement/Sections/Finances.js

```diff
@@ -22,6 +23,14 @@ export default class Finances extends React.Component {
     return <Badge data-test-finances-agreement-lines-count={count}>{count}</Badge>;
   }
 
+  renderInvoices = () => {
+    const invoices = get(this.props, 'invoices', []);
+    if (!invoices.length) {
+      return null;
+    }
+    return <div align="end"><FormattedMessage id="ui-agreements.agreements.showInvoicesLink" /></div>;
```

Comment:
Can you use the `Layout` component directly here instead of rendering `div` markup?

Here's its [readme](https://github.com/folio-org/stripes-components/tree/master/lib/Layout) and [storybook](https://ux.folio.org/storybook/?selectedKind=Layout&selectedStory=Available%20classes&full=0&addons=1&stories=1&panelRight=0&addonPanel=REACT_STORYBOOK%2Freadme%2Fpanel)

Reply:
Yes, done.